src/lemlib/odom/differentialArc.cpp

```diff
@@ -138,33 +138,36 @@ void DifferentialArc::update() {
         infoSink()->error("Odom calculation failure! Not enough sensors to calculate heading");
         return;
     }
-    const float deltaTheta = theta - pose.theta;
-    const float avgTheta = pose.theta + deltaTheta / 2;
+    const Angle deltaTheta = theta - pose.theta;
+    const Angle avgTheta = pose.theta + deltaTheta / 2;
 
     // calculate local change in position
-    Pose local(0, 0, deltaTheta);
+    Pose local(0_in, 0_in, deltaTheta);
     // set sinDTheta2 to 1 if deltaTheta is 0. Simplifies local position calculations.
-    const float sinDTheta2 = (deltaTheta == 0) ? 1 : 2 * std::sin(deltaTheta / 2);
+    const float sinDTheta2 = (deltaTheta == 0_deg) ? 1 : 2 * units::sin(deltaTheta / 2).raw();
```

Comment:
```suggestion
    const float sinDTheta2 = (deltaTheta == 0_rad) ? 1 : 2 * units::sin(deltaTheta / 2).raw();
```
Change to radians for consistency